kustomize/overlays/minikube/shard/kustomization.yaml

```diff
@@ -1,6 +1,7 @@
 secretGenerator:
```

Comment:
@vaibhavjainwiz We'll also need any required variables added to our `prod` and `stable` overlays as well. There are instructions in the `kustomize/README.md` on how to do that.

cheers,

Rob